### PR TITLE
iscan and db revised

### PR DESCRIPTION
**Pull request by mvovchuk**

### CODE REVIEW

**Review by leighvdveen — APPROVED** [review]

**Comment by mvovchuk** [comment]
cannot be merged until tested

**Comment by mvovchuk** [comment]
Leigh vd Veen  15:01 I've removed the {:target="_blank"} from the migration-db.md file - there are 2 comments - I've indicated them as \#### - they need to be removed when the code and process flow has been checked etc

File: gitlab-iscan.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+To deploy Elastio inside your CI, you need to run the following command to creat the folder that AWS reads its credentials from: 
+
+```
+mkdir ~/.aws && touch ~/.aws/config ~/.aws/credentials 
+```
+
+### Step 2 
+
+Once this command has successfully completed, the next step is to take the variables that you added to your repo and add them to the reated  `~/.aws/config ~/.aws/credentials` files as follows:
```
Comment (arybitskyi) [inline gitlab-iscan.md:25]: is this a typo "reated "?

Comment (leighvdveen) [reply gitlab-iscan.md:25]: apologies - yes - it should be "created"

File: gitlab-iscan.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        aws_access_key_id = $aws_access_key_id
+        aws_secret_access_key = $aws_secret_access_key
+      END_TEXT
+```
+### Step 3
+
+Run the following script in order to install the Elastio CLI and the Elastio Shell: 
+
+```
+- apt install unzip curl git wget -y
```
Comment (arybitskyi) [inline gitlab-iscan.md:46]: This command is Ubuntu specific. However script to install elastio is cross platform. Should we just mention that this utils are needed without instructions on how install them?

File: gitlab-iscan.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+- /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com elastio/elastio-stack/ master/scripts/install-elastio.sh)"
+- curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64 zip" -o "awscliv2.zip" && unzip awscliv2.zip && ./aws/install && apt install python3-pip -y
+- pip3 install --extra-index-url=https://dl.cloudsmith.io public/elastio/public/python/simple/  elastio-shell
+```
+
+ ### Step 4
+
+Run the follwown script to install the Integrity Scan (iscan) module:
```
Comment (arybitskyi) [inline gitlab-iscan.md:54]: Typo "follwown"

File: migration-db.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
+# Migrate a PostgreSQL Database From One Environment to Another Using Elastio
+
+A common problem, slowing down the SDLC (Software Development Life Cycles, occurs when the databases in the different environments (dev, qa, staging, and prod) are not in sync. And, without an automated process of sorts, DBAs will battle to ensure that every environment's databases are always synchronized.
```
Comment (arybitskyi) [inline migration-db.md:3]: I do not understand this sentence: "And, without an automated process of sorts"
Do we really want to use "sorts" here?

File: migration-db.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+   2. Type in the following SQL statement to create the new phonebook database (and press \<enter>):
+      ```
+      CREATE DATABASE phonebook;
+      ```
+
+   3. Run the following SQL command to import the phone_book.sql file:
+      ```
+      postgres -f <file path>/phone_book.sql phonebook
+      ```
+\#### This command isn't importing the phone_book.sql file - either the command is wrong or the sql file isn't correct.
```
Comment (arybitskyi) [inline migration-db.md:56]: This comment should be removed. 
I was unable to create and import database using commands above, but I do not have a lot of experience in postgres.
I used commands:
sudo -u postgres createdb phonebook
sudo -u postgres psql phonebook  <  <file path>/phone_book.sql

File: migration-db.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+      ```
+\#### This command isn't importing the phone_book.sql file - either the command is wrong or the sql file isn't correct. 
+
+   4. To confirm whether the phonebook database was created, select the phonebook database by running the following command: 
+      ```
+      \l 
+      ```
+      The output should be as follows: 
+
+      ![image](images/postgresql-output.png)
```
Comment (arybitskyi) [inline migration-db.md:64]: This image is missing

**Pull request closed without merging**